lib/src/sliver_value_layout_builder.dart

```diff
@@ -2,17 +2,16 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 
 /// The signature of the [SliverValueLayoutBuilder] builder function.
-typedef SliverValueLayoutWidgetBuilder<T> = Widget Function(
+typedef Widget SliverValueLayoutWidgetBuilder<T>(
```

Comment:
Why using the old syntax?
https://dart.dev/guides/language/effective-dart/design#dont-use-the-legacy-typedef-syntax